lbry/lbry/extras/cli.py

```diff
@@ -242,11 +242,11 @@ def run_daemon(args: list, conf: Config):
 
     loop = asyncio.get_event_loop()
 
-    if args.verbose:
-        log.setLevel(logging.DEBUG)
+    log.setLevel(logging.DEBUG if args.verbose == [] else logging.INFO)
+    for module in args.verbose or []:
+        logging.getLogger(module).setLevel(logging.DEBUG)
+    if isinstance(args.verbose, list):
         loop.set_debug(True)
-    else:
-        log.setLevel(logging.INFO)
```

Comment:
The conditionals and logic here is a bit convoluted and hard to follow, can you simplify/clean this up?

Reply:
Hopefully cleared it up. I can add comments or a few new variables, if you think it's necessary...